src/checkDifferences.ts

```diff
@@ -15,8 +15,8 @@ export const checkDifferences = async (shotItems: ShotItem[]) => {
   );
 
   const total = shotItems.length;
-  let differenceCount = 0;
-  let noBaselinesCount = 0;
+  const noBaselinesItems: ShotItem[] = [];
+  const diffInShotItems: ShotItem[] = [];
```

Comment:
Please let's change that to `aboveThresholdDifferenceItems` so it will be perfectly clear what it holds.

Reply:
thanks @chriskalmar for review, here's adjustment: [rename aboveThresholdDifferenceItems](https://github.com/lost-pixel/lost-pixel/pull/309/commits/6cfd0be58432f5eba3a6ce610d0730e77a1ac933)

Reply:
perfect, thanks @tn819